examples/complete/module/compare/compare.tf

```diff
@@ -0,0 +1,42 @@
+variable "a" {
```

Comment:
Consider adding an inline comment at the top of this file indicating that it is used for backwards compatibility testing.